src/Model/DocsAsHtml.hs

```diff
@@ -241,15 +240,6 @@ splitOnPathSep str
   | '\\' `elem` str = splitOn "\\" str
   | otherwise       = [str]
 
-renderFixity :: P.Fixity -> Html ()
-renderFixity (P.Fixity associativity precedence) =
```

Comment:
I think we should be able to leave this in - the information is still there, as far as I know, it's just been moved into the `DeclarationInfo`.